client/src/components/ImportComponent/ImportComponent.js

```diff
@@ -57,10 +57,19 @@ export class ImportComponent extends Component {
     await this.cyRef.current.doLayout();
   }
 
+  formatJsonNumberIntoString = data => {
+    for (const twin of data.digitalTwinsGraph.digitalTwins) {
+      if (typeof twin.$dtId === "number") {
+        twin.$dtId = twin.$dtId.toString();
+      }
+    }
+  }
```

Comment:
I think this would sit better in the import service, rather than in the view layer.

Reply:
Got it!